packages/webdriverio/src/driver/index.ts

```diff
@@ -0,0 +1,174 @@
+import fs from 'node:fs/promises'
+import url from 'node:url'
+import path from 'node:path'
+import cp from 'node:child_process'
+
+import getPort from 'get-port'
+import waitPort from 'wait-port'
+import logger from '@wdio/logger'
+import WebDriver from 'webdriver'
+// @ts-expect-error
+import { getChromeVersion } from '@testim/chrome-version'
+
+import { start as startSafaridriver, type SafaridriverOptions } from 'safaridriver'
+import { start as startGeckodriver, type GeckodriverParameters } from 'geckodriver'
+import { start as startEdgedriver, type EdgedriverParameters } from 'edgedriver'
+import { install, getInstalledBrowsers, Browser, type InstallOptions } from '@puppeteer/browsers'
+
+import type { Capabilities } from '@wdio/types'
+
+import detectBackend from './detectBackend.js'
+import { parseParams } from './utils.js'
+import { SUPPORTED_BROWSERNAMES } from '../constants.js'
+import type { RemoteOptions } from '../types'
+
+const log = logger('webdriver')
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
```

Comment:
since NodeJs already uses a global with the same name, I would change this to:
`const dirname`